package.json

```diff
@@ -42,13 +42,14 @@
     "immutable": "^4.0.0-rc.12",
     "intersection-observer": "^0.7.0",
     "lodash": "^4.17.11",
-    "manifesto.js": "^3.0.9",
+    "manifesto.js": "^3.0.11",
     "node-fetch": "^2.6.0",
     "node-sass": "^4.12.0",
     "normalize-url": "^4.2.0",
     "openseadragon": "^2.4.1",
     "prop-types": "^15.6.2",
     "react-aria-live": "^2.0.5",
+    "react-beautiful-dnd": "^11.0.4",
```

Comment:
Would it be possible to integrate with existing DnD libraries we are using? `react-rnd` or `react-dnd` (dependency of react-mosaic)

Just curious since we are using a few different DnD libraries now. However, react-beautiful-dnd seems really nice.

Reply:
Not really. `react-rnd` and `react-dnd` are pretty low-level and generic. `react-beautiful-dnd` is tailored for sorting lists of things, and so it does a bunch of the high-level functionality and provides a bunch of accessibility features for us... and, likewise, we can't replace those other libraries with it, because `react-beautiful-dnd` is pretty much exclusively about sorting lists of things.